### ISSUE TITLE
IU1.2Beta3 version mismatch

### ISSUE THREAD

**Issue by TicTac9k1** [ec-/Quake3e#214]
I try to load this mod but I'm running into the error version mismatch instaunlagged/baseq3-1

quake3.exe +set fs_game "IU1.2Beta3" - This works and loads mod fine
quake3e-vulkan.x64.exe +set fs_game "IU1.2Beta3" - This gives a version mismatch when loading into a map. It has worked fine roughly 2 years ago.

I'm at a loss where to look for the solution. Did mod compatibility for IU1.2Beta3 break somehow?

**Comment by ensiform** [comment]
This is incompatibility with cgame and qagame mod. Not related to engine.

**Comment by TicTac9k1** [comment]
I did start a listen server, and it seems like all components are there since it loaded fine with the base game, and I pulled it from https://rktclan.ucoz.net/load/1-1-0-15 since it was the only source I could find that still hosted the mod.

Although I can't remember if I was using ioquake or q3e at the time though. I knew of Q3E's existance but I'm not 100% sure what was running on my old server that so I will have to check that server and hope I have the files on there still. 

It's not the end of the world however, since I found that ExcessivePlus still works and supposedly there's a config that imitates the behaviour, though sadly barred behind a registration that seemingly doesn't work.
